aspnetcore/performance/rate-limit.md

```diff
@@ -172,63 +471,17 @@ In the preceding controller:
-
-:::code language="csharp" source="~/../AspNetCore.Docs.Samples/fundamentals/middleware/rate-limit/WebRateLimitAuth/Program.cs" id="snippet_1":::
-
-> [!WARNING]
->Creating partitions on client IP addresses makes the app vulnerable to Denial of Service Attacks which employ IP Source Address Spoofing. For more information, see [BCP 38 RFC 2827 Network Ingress Filtering: Defeating Denial of Service Attacks which employ IP Source Address Spoofing](https://www.rfc-editor.org/info/bcp38).
-
-See [the samples repository for the complete `Program.cs`](https://github.com/dotnet/AspNetCore.Docs.Samples/blob/main/fundamentals/middleware/rate-limit/WebRateLimitAuth/Program.cs#L145,L281) file.
-
-The `SampleRateLimiterPolicy` class
-
-:::code language="csharp" source="~/../AspNetCore.Docs.Samples/fundamentals/middleware/rate-limit/WebRateLimitAuth/SampleRateLimiterPolicy.cs" id="snippet_1":::
-
-In the preceding code, <xref:Microsoft.AspNetCore.RateLimiting.RateLimiterOptions.OnRejected> uses <xref:Microsoft.AspNetCore.RateLimiting.OnRejectedContext> to set the response status to [429 Too Many Requests](https://developer.mozilla.org/docs/Web/HTTP/Status/429). The default rejected status is [503 Service Unavailable](https://developer.mozilla.org/docs/Web/HTTP/Status/503).
-
-### Limiter with authorization
-
-The following sample uses JSON Web Tokens (JWT) and creates a partition with the JWT [access token](https://github.com/dotnet/aspnetcore/blob/fd1891536f27e959d14a140ff9307b6a21191de9/src/Security/Authentication/JwtBearer/src/JwtBearerHandler.cs#L152-L158). In a production app, the JWT would typically be provided by a server acting as a Security token service (STS). For local development, the dotnet [user-jwts](xref:security/authentication/jwt) command line tool can be used to create and manage app-specific local JWTs.
-
-:::code language="csharp" source="~/../AspNetCore.Docs.Samples/fundamentals/middleware/rate-limit/WebRateLimitAuth/Program.cs" id="snippet_jwt":::
-
-### Limiter with `ConcurrencyLimiter`, `TokenBucketRateLimiter`, and authorization
-
-The following sample:
-
-* Adds a `ConcurrencyLimiter` with a policy name of `"get"` that is used on the Razor Pages.
-* Adds a `TokenBucketRateLimiter` with a partition for each authorized user and a partition for all anonymous users.
-* Sets [RateLimiterOptions.RejectionStatusCode](xref:Microsoft.AspNetCore.RateLimiting.RateLimiterOptions.RejectionStatusCode) to [429 Too Many Requests](https://developer.mozilla.org/docs/Web/HTTP/Status/429).
+## Rate limiting metrics
 
-:::code language="csharp" source="~/../AspNetCore.Docs.Samples/fundamentals/middleware/rate-limit/WebRateLimitAuth/Program.cs" id="snippet_adm2":::
+The rate limiting middleware provides built-in metrics and monitoring capabilities to help understand how rate limits are affecting app performance and user experience. The following metrics are provided for rate limiting
```

Comment:
Link to https://learn.microsoft.com/aspnet/core/log-mon/metrics/metrics?

Reply:
The names provided below aren't correct.
See https://learn.microsoft.com/dotnet/core/diagnostics/built-in-metrics-aspnetcore?view=aspnetcore-9.0#microsoftaspnetcoreratelimiting

Reply:
> The names provided below aren't correct. See https://learn.microsoft.com/dotnet/core/diagnostics/built-in-metrics-aspnetcore?view=aspnetcore-9.0#microsoftaspnetcoreratelimiting

I deleted that table and added a link to `#microsoftaspnetcoreratelimiting`